.env

```diff
@@ -39,7 +39,7 @@ OPENPLATFORM_SEARCH_LIMIT=100
 
 # Prefix used to match index template
 ELASTIC_STATS_INDEX_PREFIX=stats
-ELASTIC_STATS_INDEX=$(date +%d-%m-%Y)
+ELASTIC_STATS_INDEX='$(date +%d-%m-%Y)'
```

Comment:
This should not be in '' as it's a subcommand thats run when the file i parsed evaluated.